--- a/arch/arm/boot/dts/broadcom/bcm4708-linksys-ea6300-v1.dts	2026-01-17 21:24:49.562467000 -0800
+++ b/arch/arm/boot/dts/broadcom/bcm4708-linksys-ea6300-v1.dts	2026-01-17 21:38:42.449992000 -0800
@@ -21,6 +21,15 @@
 		reg = <0x00000000 0x08000000>;
 	};
 
+ 	leds { 
+ 	       compatible = "gpio-leds"; 
+ 	       power { 
+ 	            label = "bcm53xx:green:power"; 
+ 	            gpios = <&chipcommon 8 GPIO_ACTIVE_HIGH>; 
+ 	            linux,default-trigger = "default-on"; 
+ 	       }; 
+ 	}; 
+
 	nvram@1c080000 {
 		compatible = "brcm,nvram";
 		reg = <0x1c080000 0x180000>;

--- a/arch/arm/boot/dts/broadcom/bcm4708-linksys-ea6500-v2.dts	2026-01-17 21:24:49.562467000 -0800
+++ b/arch/arm/boot/dts/broadcom/bcm4708-linksys-ea6500-v2.dts	2026-01-17 21:49:43.810868000 -0800
@@ -23,6 +23,15 @@
 		      <0x88000000 0x08000000>;
 	};
 
+ 	leds { 
+ 	       compatible = "gpio-leds"; 
+ 	       power { 
+ 	            label = "bcm53xx:green:power"; 
+ 	            gpios = <&chipcommon 8 GPIO_ACTIVE_HIGH>; 
+ 	            linux,default-trigger = "default-on"; 
+ 	       }; 
+ 	}; 
+
 	gpio-keys {
 		compatible = "gpio-keys";
 
